src/components/admin/technical-maintenance-tab.yml

```diff
@@ -0,0 +1,33 @@
+tabInfo:
+  en: You can trigger different tasks from here, in order to execute technical maintenance over the database and the CDN.
+  de: You can trigger different tasks from here, in order to execute technical maintenance over the database and the CDN.
+documentsValidationHeader:
+  en: Documents validation
+  de: Dokumente Validierung
```

Comment:
In "nominal" form:

Validierung der Dokumente / Neu-Erstellung der Dokumente / Konsolidierung der CDN-Ressourcen / Erstellung der CDN-Verzeichnisse

Or in "infinitive" form:

Dokumente validieren / Dokumente neu erstellen / CDN-Ressourcen konsolidieren / CDN-Verzeichnisse erstellen

In German to me the "infinitive" form sounds more natural while the "nominal" form sounds a little bit artificial, don't know about the English, though.

Reply:
Thanks alot! 🙏 Changed everything as you wrote it.